src/client/datascience/jupyter/kernels/helpers.ts

```diff
@@ -1770,7 +1780,16 @@ export async function executeSilently(session: IJupyterSession, code: string): P
         }
     };
     await request.done;
-    traceInfo(`Executing silently Code (completed) = ${code.substring(0, 100).splitLines().join('\\n')}`);
 
+    const codeForLogging = `(Code = ${code.substring(0, 100).splitLines().join('\\n')})`;
+    if (errorMessageForLogging && errorMessages.length) {
```

Comment:
If we have an error message, then log failure messages as errors, else as warnings.